packages/lib/src/types/alluvial.ts

```diff
@@ -7,7 +7,7 @@ import {
 } from 'd3-sankey';
 
 import { Color } from '@/utils/constants';
-import { Dataset, DatasetValueObject } from '@/types/dataset';
+import type { Dataset, DatasetValueObject } from '@/types/dataset';
```

Comment:
Have you checked for circular dependencies? I think some of these imports might cause them 🤔

Reply:
Good point!
Used `madge` to check for circular dependencies - we don't have any :tada:

<img width="594" alt="Screenshot 2024-04-03 at 11 08 24" src="https://github.com/Adyen/lume/assets/8277926/d41f1ede-27db-4aad-b1e0-ea6faabd7d9a">

Reply:
Amazing!